pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/kit/KitDaoImpl.java

```diff
@@ -583,4 +587,17 @@ public Optional<ScanError> updateKitLabel(KitRequestShipping kitRequestShipping)
         }
         return result;
     }
+
+    public boolean hasKitReceived(Connection connection, String ddpParticipantId) {
```

Comment:
This method looks like something very generic. Something like `contains` or `exists`. I would suggest defining it somewhere separately and reusing it wherever we do similar operations.